### PR TITLE
Fix issue 1031 and problem with testRigidTransform2D

### PR DESCRIPTION
**Pull request by JacquesOlivierLachaud**
Fix issue #1031 (include and eigen version).
Also remove testRigidTransform2D for now since it creates an infinite pgm file.

### CODE REVIEW

**Comment by JacquesOlivierLachaud** [comment]
Failed CI builds are unrelated to the PR. I think it can be merged.

**Comment by dcoeurjo** [comment]
@JacquesOlivierLachaud could you please merge the master to this branch ?

**Comment by JacquesOlivierLachaud** [comment]
Done.

File: src/DGtal/math/linalg/SimpleMatrix.ih [hunk 1]
```diff
@@ -59,11 +59,13 @@ DGtal::SimpleMatrix<T,TM, TN>::SimpleMatrix()
...
   for ( DGtal::Dimension i = 0; i < TM*TN; ++i )
     myValues[ i ] = NumberTraits<Component>::ZERO;
-
+  const Component one       =  NumberTraits<Component>::ONE;
+  const Component minus_one = -NumberTraits<Component>::ONE;
   //Cofactor coefs computation
   for (DGtal::Dimension i=0; i<TM; i++)
     for (DGtal::Dimension j=0; j<TN; j++)
-      myCofactorCoefs[i*N+j] = static_cast<T>(std::pow(-1.0f, (float)i+j));
+      myCofactorCoefs[i*N+j] = ( (i+j) & 1 )
```
Comment (dcoeurjo) [inline src/DGtal/math/linalg/SimpleMatrix.ih]: is "&" operator defined if Component is GMP integers ?

Comment (dcoeurjo) [inline src/DGtal/math/linalg/SimpleMatrix.ih]: my bad I'm wrong.. is it surely defined on DGtal::Dimension

**Comment by dcoeurjo** [comment]
ok thanks.. merging

**Pull request merged**